Procfile

```diff
@@ -1,4 +1,5 @@
 release: python manage.py migrate --noinput
 web: gunicorn cardboard.wsgi --log-file -
 worker: celery -A cardboard worker -l INFO --without-heartbeat --without-gossip --without-mingle --concurrency 2
+worker: celery -A cardboard worker -l INFO --without-heartbeat --without-gossip --without-mingle --concurrency 1 -Q activity_updates_queue --beat
```

Comment:
Does this worker for the Google Drive activity run on the same dyno as the other worker? Just curious if this requires another dyno. For saving money, it'd be nice if it could run in the same dyno as the other worker.

Reply:
Different dyno, I think it's better for it to be separate, at least this year, since this is a feature that might block more important things that the other celery workers are doing, like sheets creation.

Reply:
i think our celery setup is pretty wasteful since all of the dynos should be idle almost all the time, but we can probably work on that separately